components/camel-xchange/src/main/docs/weather-component.adoc

```diff
@@ -0,0 +1,176 @@
+== Weather Component
```

Comment:
Looks like the docs are from the weather component

Reply:
I updated the docs with an initial draft. Some of the sections seem to be generated & I would need a little help on how to do this. Also, I'd like to use the name part of the URI, which for some reason in null in the configuration. Perhaps you could have a look at what I might have done wrong here.

Reply:
Is there a way to mark the component as preview? The API is still expected to change a lot while I move along.

Reply:
Actually you need to add a comment manually for the preview status. At first glance to generate the docs automatically with component and endpoint options, you need to name the .adoc with the component name, so xchange-component.adoc.